.storybook/main.js

```diff
@@ -6,17 +6,25 @@ module.exports = {
     '@storybook/addon-interactions',
     '@storybook/addon-a11y'
   ],
-  framework: '@storybook/react',
-  core: {
-    builder: '@storybook/builder-vite'
+  framework: {
+    name: '@storybook/react-vite',
+    options: {}
   },
   features: {
     storyStoreV7: true
   },
   async viteFinal(config, { configType }) {
     config.base = process.env.BASE_PATH || config.base;
 
+    // Skip type declaration generation for non-dist builds
+    config.plugins = config.plugins.filter(
+      plugin => plugin.name !== 'vite:dts'
+    );
+
```

Comment:
This plugin was blocking Netlify deployments.  Since we're only removing its execution in the Storybook config, this should not affect distribution builds.